velox/functions/prestosql/tests/SplitToMapTest.cpp

```diff
@@ -80,6 +80,63 @@ TEST_F(SplitToMapTest, basic) {
   tryCallFunc(data);
 }
 
+TEST_F(SplitToMapTest, MultiCharDelim1) {
+  auto callFunc = [&](const RowVectorPtr& input) {
+    return evaluate("split_to_map(c0, ',', ';=')", input);
+  };
+
+  // Valid string cases.
+  auto data = makeRowVector({
+      makeFlatVector<std::string>({
+          "1;=10,2;=20,3;=30,4;=",
+          "4;=40,",
+          "",
+          ";=00,1;=11,3;=33,5;=55,7;=77",
+          ";=",
+      }),
+  });
+
+  auto result = callFunc(data);
+
+  auto expected = makeMapVector<std::string, std::string>(
+      {{{"1", "10"}, {"2", "20"}, {"3", "30"}, {"4", ""}},
+       {{"4", "40"}},
+       {},
+       {{"", "00"}, {"1", "11"}, {"3", "33"}, {"5", "55"}, {"7", "77"}},
+       {{"", ""}}});
+
+  velox::test::assertEqualVectors(expected, result);
+}
+
+TEST_F(SplitToMapTest, MultiCharDelim2) {
```

Comment:
Can you add to cover the invalid case with the two multi chars? Like could not found a valid entry after the entry delimiter or the  value entry after the key/value entry delimiter? Thanks!